sdk/devcenter/azure-developer-devcenter/CHANGELOG.md

```diff
@@ -1,15 +1,25 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-02-07)
+
+This release updates the Azure DevCenter library to use the 2022-11-11-preview API.
 
 ### Features Added
 
-### Breaking Changes
+- `DevBoxClient`, `DevCenterClient`, and `EnvironmentsClient` now accept an endpoint URI on construction rather than tenant ID + dev center name.
+- Added upcoming actions APIs to dev boxes.
+    - `delayUpcomingActionWithResponse`
+    - `getUpcomingActionWithResponse`
+    - `listUpcomingActions`
+    - `skipUpcomingActionWithResponse`
 
 ### Bugs Fixed
+- Invalid `beginDeleteEnvironmentAction` API removed from `EnvironmentsClient`.
 
 ### Other Changes
 
+- Unimplemented artifacts APIs removed from `EnvironmentsClient`.
```

Comment:
Understand that you're going to improve changelog.

---

Should this be in "### Breaking Changes"?

Also need to mention the change in Builder there as well, as it breaks user.

Reply:
Yep! Sorry will correct that

Reply:
Actually RE: artifacts APIs - while this is technically a breaking change, the artifacts methods in the API should have never worked because the endpoints were never implemented in the Data Plane. Really I would consider it something more like a bug fix I guess, similar to removing the invalid `beginDeleteEnvironmentAction`. Does that sound fair?

Reply:
Fine for me.